algosdk/transaction.py

```diff
@@ -520,6 +520,18 @@ def __eq__(self, other):
             and self.sprfkey == other.sprfkey
         )
 
+    @staticmethod
+    def fixed_bytes64(key, size):
```

Comment:
Very minor, but prefixing this with `_` would clearly mark this as an internal function

Reply:
ok, I'll hold my nose and do it.

leading underscore to fake privacy in a language that doesn't have it is one my least favorite traditions.